requirements.txt

```diff
@@ -14,7 +14,7 @@ html5lib==1.1
 idna==3.7
 importlib_metadata==7.1.0
 ipykernel==6.29.3
-ipython==8.24.0
+ipython==7.34.0
```

Comment:
Do we still want to revert this back to this number if we don't think that's what's causing the issue? I'd vote to leave this change out but open to input